addons/storyshots/stories/directly_required/Button.js

```diff
@@ -1,28 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
In theory this file is testing different ways of requiring stories in storyshots.

In practice we don't exercise this test right now. I'm not sure how to do it :/

In any case, I'm not sure we should remove it yet?